contracts/JBVeNft.sol

```diff
@@ -456,6 +456,14 @@ contract JBVeNft is IJBVeNft, veERC721, Ownable, ReentrancyGuard, JBOperatable {
       // Burn the token.
       _burn(_data.tokenId);
 
+
+      // giving permission to the terminal to burn the locked jb tokens
+      uint256[] memory _permissionIndexes = new uint256[](1);
+      _permissionIndexes[0] = JBOperations.BURN;
+      operatorStore.setOperator(
+      JBOperatorData(address(_data.terminal), projectId, _permissionIndexes)
```

Comment:
Isn't this risky? Since a user could call this method with the `_data.terminal` being their own (malicious) contract, then they have access to burn all of the tokens this contract holds

Reply:
Hey @xBA5ED 
since the contract is the owner of the locked tokens so we don't need to give permission
thanks for the spot let me know if everything looks okay to you

Reply:
> I did discover a burn permission issue sometime back wen I was testing this but now it looks okay removing this wdyt ?

Do you remember what caused/fixed it?

Reply:
I think either there was some issue in tests wen i was debugging or I was using the old jbx contract version